README.md

```diff
@@ -58,6 +58,7 @@ This can be chosen using the input `validator_version`.
 | Input | Description | Usage | Default | Action version |
 | :---: |    :---     | :---: |  :---:  |      :---:     |
 | `all_versioned_paths` | Whether to test all possible versioned base URLs:<br><br>/vMAJOR<br>/vMAJOR.MINOR<br>/vMAJOR.MINOR.PATCH<br><br>The latter two being optional base URLs according to the specification. | Optional | `false` | `v1+`
+| `validate_unversioned_path` | Whether to validate the input URL as a full OPTIMADE implementation without appending any version.<br><br>This action assumes that the 'path' parameter will contain an unversioned URL. As it is not mandatory to run a full OPTIMADE implementation from the unversioned URL, by default, this action will not perform any validation of this URL, beyond checking the `/versions` endpoint. | Optional | `false` | `v2.3+` 
```

Comment:
> > FYI, I have written the README expecting there to be a v2.3.0 release immediately after this PR.
> 
> No need. This will be updated by the publishing workflow (via the `invoke` task).

Even this bit? Can revert the line above though

Reply:
No, not this bit. Only the standard places where versions are mentioned.
Also, it doesn't work for new major versions (as far as I remember).